README.md

```diff
@@ -54,7 +54,133 @@ hasp_lvgl:
               message: Hello {{ text }}
 ```
 
-in the event service call you can use any variable coming from the MQTT message besides the actual event
+In the event service call any variable coming from the MQTT message can be used between curly brackets. 
+
+### Example 1: cover control with state feedback
+![kép](https://user-images.githubusercontent.com/1550668/112142921-77daf580-8bd7-11eb-9626-ebfb3423629d.png)
```

Comment:
what's kép ?

Reply:
it's generated by the browser when I Ctrl+V the picture from clipboard